src/components/MapView/MapView.tsx

```diff
@@ -42,31 +56,23 @@ const MapView = forwardRef<MapViewHandle, ComponentProps>(
                     return;
                 }
 
+                if (!shouldPanMapToCurrentPosition()) {
+                    setCurrentPositionToInitialState();
+                    return;
+                }
+
```

Comment:
Why don't we just return early?

Reply:
@s77rt It is because `currentPosition` needs to bet set so that pending view is not displayed.
https://github.com/Expensify/App/blob/6f3d8f6f738a098a3628dd604e27d45d53d368ef/src/components/MapView/MapView.tsx#L145